### PR TITLE
This is a part of Implement in Java #1999

### PR DESCRIPTION
**Pull request by Amit366**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

### Description

Implementation of Stack using Queue.

### CODE REVIEW

**Comment by HarshCasper** [comment]
Closing the PR due to no Issues. Please add an Issue to your PRs, so that we can take a look, at the implementation you are looking to work on. 

If you are working on an Issue, add the link here and refer me here/or on Discord. I'll take a look.

**Comment by Amit366** [comment]
@HarshCasper sir for the implementation in Java issue #1999 there are 4 parts out of which I have sent 2 . So this is one of the parts. So please consider.

**Comment by Amit366** [comment]
https://github.com/TesseractCoding/NeoAlgo/issues/1999 

This is the issue

**Comment by Amit366** [comment]
@HarshCasper sir it's 10 days since the PR is sent. Only few days are left for the competition. So please merge it.

File: Java/cp/Stack_using_Queue.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
+
+/* Java Program to implement a stack using two queue */
```
Comment (HarshCasper) [inline Java/cp/Stack_using_Queue.java:2]: Add a description about the Code/Algorithm ?

File: Java/cp/Stack_using_Queue.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+        int n = sc.nextInt();
+        for(int i = 1; i <= n; i++)  //accepting input from user
+          s.push(i); 
+        
+        for(int i = 1; i <= n; i++){
+           System.out.println(s.top()); //printing the output
+           s.pop(); 
+    }
+  
+    }
```
Comment (HarshCasper) [inline Java/cp/Stack_using_Queue.java]: Format them properly

File: Java/cp/Stack_using_Queue.java [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+  
+    // driver code 
+    public static void main(String[] args) 
+    { 
+        Scanner sc = new Scanner(System.in);
+        Stack s = new Stack(); 
+
+        System.out.println("Enter the limit");
+        int n = sc.nextInt();
+        for(int i = 1; i <= n; i++)  //accepting input from user
```
Comment (HarshCasper) [inline Java/cp/Stack_using_Queue.java]: Keep the Comments above the Line of Code

File: Java/cp/Stack_using_Queue.java [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+          s.push(i); 
+        
+        for(int i = 1; i <= n; i++){
+           System.out.println(s.top()); //printing the output
```
Comment (HarshCasper) [inline Java/cp/Stack_using_Queue.java]: Keep the Comments above the Line of Code

**Review by HarshCasper — CHANGES_REQUESTED** [review]
Remove Extra Lines of Code

File: Java/cp/Stack_using_Queue.java (hunk 1 above)
Comment (Amit366) [reply Java/cp/Stack_using_Queue.java:2]: I have given the description. 
Java Program to implement a stack using two queue */

**Comment by Amit366** [comment]
@HarshCasper Sir please merge it now.

**Review by atarax665 — APPROVED** [review]

**Comment by Amit366** [comment]
@HarshCasper sir please merge it

**Comment by Amit366** [comment]
@HarshCasper sir please add the respective level tag

**Comment by Amit366** [comment]
@atarax665 Sir it was a level 3 issue.

**Comment by Amit366** [comment]
Level 2 are for minor changes but a program is a level 3

**Pull request merged**